task4/func.js

```diff
@@ -0,0 +1,21 @@
+function Person(name,age) {
+    this.name = name;
+    this.age = age;
+}
+
+Person.prototype.introduce = function () {
+    return 'My name is ' + this.name + ' and I am ' + this.age;
+}
+
+function myNew(type, ...args) {
```

Comment:
type is a little bit strange name :)
Maybe constructorName or className will be more meaningful

Reply:
Your `myNew` method will work only with the objects, but as you know we could create arrays and functions via `new`. The task asks for `myNew` function to act as the keyword new, and that means that you need to care about these cases also.
Please, carefully read the article with a description of new operator
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/new#description
It will help you :)